src/components/TableBody/TableBody.tsx

```diff
@@ -0,0 +1,94 @@
+import { useMemo } from 'react';
+import classNames from 'classnames';
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { Person } from '../../types/Person';
+
+interface Props {
+  people: Person[];
+  filteredPeople: Person[]
+}
+
+export const TableBody: React.FC<Props> = ({ people, filteredPeople }) => {
+  const { slug } = useParams();
+  const { search } = useLocation();
+
+  const listOfNames = useMemo(
+    () => people.map(person => person.name),
+    [people],
+  );
+
+  const findSlugOfChoosedPerson = (choosedPerson: string) => {
+    return people.find(person => person.name === choosedPerson)?.slug;
+  };
+
+  return (
+    <tbody>
+      {filteredPeople.map(person => {
+        const {
+          name,
+          sex: personSex,
+          born,
+          died,
+          fatherName,
+          motherName,
+          slug: personSlug,
+        } = person;
+
+        return (
+          <tr
+            key={personSlug}
+            data-cy="person"
+            className={classNames({
+              'has-background-warning': personSlug === slug,
+            })}
+          >
+            <td>
+              <Link
+                relative="path"
+                to={`../${personSlug}${search}`}
+                className={classNames({
+                  'has-text-danger': personSex === 'f',
+                })}
+              >
+                {name}
+              </Link>
+            </td>
+
+            <td>{personSex}</td>
+            <td>{born}</td>
+            <td>{died}</td>
+
+            {motherName
```

Comment:
```suggestion
            {!!motherName
```
in rendering with &&  need to convert the value to boolean.